### PR TITLE
Update fa.js week start day

### PR DESCRIPTION
**Pull request by amirsafiallah**
`weekStartsOn` should be `6` since week starts on Saturday in Persian.

### CODE REVIEW

File: src/locale/fa.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const locale = {
   weekdays: 'یک‌شنبه_دوشنبه_سه‌شنبه_چهارشنبه_پنج‌شنبه_جمعه_شنبه'.split('_'),
   weekdaysShort: 'یک\u200cشنبه_دوشنبه_سه\u200cشنبه_چهارشنبه_پنج\u200cشنبه_جمعه_شنبه'.split('_'),
   weekdaysMin: 'ی_د_س_چ_پ_ج_ش'.split('_'),
+ weekStartsOn: 6,
```
Comment (iamkun) [inline src/locale/fa.js]: the key name shoud be 'weekStart'

Comment (amirsafiallah) [reply src/locale/fa.js]: my apologies

**Comment by iamkun** [comment]
Thanks.

**Pull request merged**